R/test-threads.R

```diff
@@ -4,11 +4,11 @@
 #' @param .threads Integer vector of threads values to test.
 #' @param .bbi_args a named list.
 #' @param .max_eval Max number of iterations for NONMEM to run.
-#'         Will update `MAXITER` or `NITER` (whichever is specified) in generated models.
+#'         Will update `MAXITER`, `NITER`, and `NBURN` (whichever is specified/found) in generated models.
```

Comment:
Not new to this PR, but this should say `MAXEVAL` not `MAXITER`, right?

Reply:
ooh Idk how I missed that. Yeah good catch